app/assets/javascripts/discourse/app/templates/signup.hbs

```diff
@@ -58,7 +58,9 @@
               />
             {{else}}
               <span class="more-info" id="account-email-validation-more-info">
```

Comment:
can we avoid the empty `<span></span>` like in the other cases (put it inside the conditional)?